praw/objector.py

```diff
@@ -151,13 +155,17 @@ def objectify(
                 return data
             if "things" in data["json"]["data"]:  # Submission.reply
                 return self.objectify(data["json"]["data"]["things"])
+            if "rules" in data["json"]["data"]:
```

Comment:
Ugh I hate this code with a passion. To be clear not your code, just the need to do all these custom parsing.